doc/src/manual/metaprogramming.md

```diff
@@ -263,29 +263,28 @@ julia> eval(ans)
 julia> ex = :(a + b)
 :(a + b)
 
-julia> eval(ex)
+julia> eval(current_module(), ex)
```

Comment:
Yeah probably. It's not neeed in the REPL but since the doctests are run in a separate module I think it is needed.